doc/design.md

```diff
@@ -53,7 +53,11 @@ The `ConfigurationParser` parses Exasol's cluster configuration format.
 
 ## `Bucket`
 
-The `Bucket` building block controls interaction with a bucket in BucketFS. 
+The `Bucket` building block controls interaction with a bucket in BucketFS.
+
+## `DatabaseService`
+
+The `DatabaseService` controls features of a service providing the database like stopping and restarting.
```

Comment:
`Restart` should be named to `start` since the implementation does not perform restart: stop -> start.

Reply:
Fixed everywhere.